### ISSUE TITLE
Missing configs

### ISSUE THREAD

**Issue by chris522229197** [eric-ai-lab/CPL#4]
Thanks for the great work! I am interested in reproducing some experimental results.

It seems like the `configs` directory is missing, so config files such as `configs/trainers/CoCoOpcf/vit_b16_c4_ep10_batch1_ctxv1.yaml` cannot be found.

It would be appreciated if all the config files for the experiments are uploaded to this repo.

**Comment by jkooy** [comment]
Hi, many thanks for your interests. They can be found in the repo here [configs/trainers/CoCoOp/vit_b16_c4_ep10_batch1_ctxv1.yaml](https://github.com/KaiyangZhou/CoOp/blob/main/configs/trainers/CoCoOp/vit_b16_c4_ep10_batch1_ctxv1.yaml)

**Comment by chris522229197** [comment]
Just making sure, does this mean that the CPL training configuration files are the same as the CoCoOp training configuration files?

**Comment by jkooy** [comment]
Yes. But CoCoop only has classification. Also specify COCOOPCF for CPL. Batch size may be adjusted in some experiments. we tested smaller ones on some machines for memory issues, this shouldn't affect the results.

**Comment by alice-cool** [comment]
> Just making sure, does this mean that the CPL training configuration files are the same as the CoCoOp training configuration files?

Do you run this code successfully?
